specification/automation/resource-manager/Microsoft.Automation/stable/2015-10-31/schedule.json

```diff
@@ -418,11 +418,38 @@
         "Day",
         "Hour",
         "Week",
-        "Month"
+        "Month",
+        "Minute"
       ],
       "x-ms-enum": {
         "name": "ScheduleFrequency",
-        "modelAsString": true
+        "modelAsString": true,
+        "values": [
+          {
+            "value": "OneTime",
+            "description": ""
```

Comment:
I think you can omit the empty description elements.

Reply:
Removed